### DISCUSSION TITLE
[Prettier Plugin] It does not work with a plain HTML project

### DISCUSSION THREAD

**Discussion by mybearworld** [tailwindlabs/tailwindcss#11580] [Help] [answered]
```json
{
  "devDependencies": {
    "prettier": "^3.0.0",
    "prettier-plugin-tailwindcss": "^0.3.0",
    "tailwindcss": "^3.3.2"
  }
}
```
Prettier's doing its thing correctly, but prettier-plugin-tailwindcss just is not working. Why?

**Comment by ines-b** [comment, marked as answer]
Have you modified the `prettier.config.js` file?

```
// prettier.config.js
module.exports = {
  plugins: [require('prettier-plugin-tailwindcss')],
  tailwindConfig: './styles/tailwind.config.js',
}
```

**Reply by mybearworld** [reply]
Shouldn't it autoload? But I'll try that later.

**Reply by mybearworld** [reply]
This errors with "TypeError: Cannot read properties of undefined (reading 'parsers')". At least it's loading now, though!

**Reply by mybearworld** [reply]
It just magically worked again when using this configuration...

**Comment by mybearworld** [comment]
It doesn't work with PostCSS either.
